apps/api/src/common/repositories/opensearch.repository.ts

```
@@ -38,38 +39,51 @@
             },
           },
         },
-      },
-    });
+      });
+      this.logger.log(
+        `Index created successfully: ${JSON.stringify(response.body, null, 2)}`,
+      );
+    } catch (error) {
+      this.logger.log(`Error creating index: ${error}`);
+      if (error.meta && error.meta.body) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)